### PR TITLE
{tools}[system/system] IntelDAAL v2019.4.007

### PR DESCRIPTION
**Pull request by robqiao**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/IntelDAAL/IntelDAAL-2019.4.007.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/intel/daal/releases/download/%(version_major)s_u%(version_minor)s']
+sources = ['l_daal_oss_p_%(version)s.tgz']
+checksums = ['bc1e521678443ed885c99f8b257d8db31e400e47ffe00c9b3fb0e80f1ad39119']
+
+extract_sources = True
+
+daal_dir = "linux/daal"
+tbb_dir = "linux/tbb"
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/i/IntelDAAL/IntelDAAL-2019.4.007.eb]: @robqiao these local variable names should now be prefixed with, for instance, `local_`, to distinguish them from easyconfig parameters (see https://easybuild.readthedocs.io/en/latest/Easyconfig-files-local-variables.html)

Comment (robqiao) [reply easybuild/easyconfigs/i/IntelDAAL/IntelDAAL-2019.4.007.eb]: Thx @migueldiascosta 👍 Fixed.

**Comment by robqiao** [comment]
@smoors can we merge this PR. Thx

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/598672083f1fe504d51b22f4413a40cb for a full test report.

**Comment by robqiao** [comment]
@smoors can we merge this PR, thx

**Comment by robqiao** [comment]
@verdurin can we merge this PR, thx

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/2d9b17e6951f794cdcefc393469e8baa for a full test report.

**Comment by Micket** [comment]
Retrigger stuck github actions

**Comment by Micket** [comment]
Going in, thanks @robqiao!

**Pull request merged**